src/GraphQL.DataLoader.Tests/GraphQL.DataLoader.Tests.csproj

```diff
@@ -21,11 +21,11 @@
     <PackageReference Include="Bogus" Version="34.0.1" />
     <PackageReference Include="Microsoft.Extensions.DependencyInjection" Version="3.1.20" />
     <PackageReference Include="Nito.AsyncEx.Context" Version="5.1.2" />
+    <PackageReference Include="System.Reactive.Linq" Version="5.0.0" />
```

Comment:
Still needed?

Reply:
Yes, the tests rely on them.  For instance you can easily create an observer that stores up events and can replay them to the test runner.